utils/ExampleRunner/example-info.json

```diff
@@ -224,6 +224,10 @@
       "advancedColorBar": {
         "name": "Advanced Colorbar",
         "description": "Demonstrates how to add an interactive colobar to stack and volume viewports with PT/CT volumes"
+      },
+      "livewire": {
+        "name": "Livewire",
```

Comment:
should be called livewireContour, i tried it it doesn't work

Reply:
Updated